modules/core/src/main/java/org/apache/ignite/configuration/PlatformNearCacheConfiguration.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.configuration;
+
+import org.apache.ignite.internal.util.typedef.internal.S;
+
+import java.io.Serializable;
+
+/**
+ * Platform near cache configuration.
+ * <p>
+ * Additional near caching mechanism on platform side (.NET).
+ */
+public class PlatformNearCacheConfiguration implements Serializable {
```

Comment:
Do we really need this class?
From my understanding, it is reserved for other platforms, otherwise, we could just send a raw byte array between .NET nodes.

Reply:
Why not? To me it looks like a good idea to allow user to configure this from xml.

Reply:
Agree with @isapego . Another thing is that we can't (trivially) avoid this, because configuration needs to be sent across entire cluster, so this has to be a part of `CacheConfiguration`.

Relevant dev list thread: http://apache-ignite-developers.2346864.n4.nabble.com/NET-Near-Cache-new-flag-in-NearCacheConfiguration-java-td45961.html